src/helperFunctions/compare_sets.py

```diff
@@ -10,14 +19,37 @@ def remove_duplicates_from_unhashable(unhashable_list: List[T]) -> List[T]:
 
 
 def remove_duplicates_from_list(list_):
+    '''
+    Remove duplicates from a list (by converting it to a set and back).
+
+    .. caution::
+        Only works with lists of hashable objects!
+
+    :param list_: the input list (possibly) containing duplicates
```

Comment:
We don't lint these (yet), but from what I've read, ending on punctuation is sort of best practice.

Reply:
changed and also added capital letters at the beginning